python/lsst/ts/Dome/mock_controller.py

```diff
@@ -448,24 +489,31 @@ async def config(self, system, settings):
                     # DM-25758: All param values are passed on as arrays so in
                     # these cases we need to extract the only value in the
                     # array.
-                    setattr(self.amcs.amcs_limits, field["target"], field["setting"][0])
+                    setattr(
+                        self.amcs.amcs_limits, field["target"], field["setting"][0],
+                    )
         elif system == LlcName.LWSCS.value:
             for field in settings:
                 if field["target"] in ("jmax", "amax", "vmax"):
                     # DM-25758: All param values are passed on as arrays so in
                     # these cases we need to extract the only value in the
                     # array.
                     setattr(
-                        self.lwscs.lwscs_limits, field["target"], field["setting"][0]
+                        self.lwscs.lwscs_limits, field["target"], field["setting"][0],
```

Comment:
The comma originally was added by black because the line was split over multiple lines. Then I renamed "lwscs_status" to "lwscs" which made the line fit on one line and black did not remove the comma. I have removed this one and one a few lines back manually and black doesn't add them back.